deploy/deploy_citus_server.yml

```diff
@@ -84,7 +85,9 @@
         port: "{{ pp_serverport }}"
         db: "{{ item }}"
         script: CREATE EXTENSION IF NOT EXISTS citus
+      register: result
       check_mode: no
+      changed_when: '"already exists" not in result.notices'
```

Comment:
Should we also check the return code (`reesult.rc`)?

Reply:
Unfortunately, the return code doesn't contain useful information, it is always 0 if the query succeeded in running at all, regardless of whether the query actually did anything.